### ISSUE TITLE
Add Continue with Microsoft button to login page

### ISSUE THREAD

**Issue by BOHEUS** [twentyhq/twenty#8472]
Right now, there's only button for logging with Google on login page but since we allow to connect Twenty with Microsoft, there should be "Continue with Microsoft" button as well.
![image](https://github.com/user-attachments/assets/ae7be6af-e3ba-4cce-af69-6e52593977ed)

For reference, that's how it looks in Settings > Accounts > Link new account
![image](https://github.com/user-attachments/assets/16098b2f-128e-44cb-beae-1d7502621187)

**Comment by AbishaKugathasan** [comment]
Hi, could I please be assigned this issue?

**Comment by BOHEUS** [comment]
@Bonapara correct me if I'm wrong but it looks like this issue can be closed now since there's additional button on login page
![Image](https://github.com/user-attachments/assets/e9d3f1cc-2ae0-409e-810c-5f293117c5ad)

**Comment by Bonapara** [comment]
@AMoreaux, can this be closed?

**Comment by Bonapara** [comment]
Please reopen if needed

**Comment by AMoreaux** [comment]
It depends if you're on the default domain so `app.twenty.com` we choose the auth methods to display. We can add Microsoft if you want

If you are on a workspace subdomain the auth methods are configured by users.

The question is, by default do you want to display the Microsoft connect?

**Comment by Bonapara** [comment]
Yes we want but we first want the Microsoft integrations (Calendar and email) to be fully implemented
